observer/lib/observer.js

```diff
@@ -1,13 +1,14 @@
 import { updateDailyTransferStats } from './platform-stats.js'
+import * as Sentry from '@sentry/node'
```

Comment:
FWIW, AFAICT, the observer bin script is not initalising Sentry - see `observer/bin/spark-observer.js`.

Fixing that is out of the scope of this pull request, though.

Reply:
Added it anyway 👼